### DISCUSSION TITLE
Help running the code

### DISCUSSION THREAD

**Discussion by Gavin5633** [yihong0618/bilingual_book_maker#202] [General]
Hello! What are the chances that you can create a video walking through how you setup this environment and run the code? No worries if not, but I am a complete novice at running code, and would love to see how to do this step by step. I do have a general idea of how to run this, but I continue running into errors. I have Anaconda and Python installed. 

This is a great solution and wanted to help my dad translate a book for him to read in Chinese.

**Comment by Gavin5633** [comment]
Any general tips our guide resources would be a nice option as well.

**Comment by yihong0618** [comment]
thanks but I do not know how to make a video when someone did it I will info you

**Reply by Gavin5633** [reply]
Appreciate this. Again, this is an amazing solution. Keep up the good work. This brings an immense amount of value.

**Comment by yihong0618** [comment]
@Gavin5633 ~https://www.bilibili.com/video/BV1XX4y1d75D
